webapp/sass/layout/_sidebar-left.scss

```diff
@@ -206,6 +206,9 @@
     }
 
     @import "../components/status-icon";
+    .status-wrapper {
+        z-index: 999999;
```

Comment:
I used this to make the dropdown accessible to the cursor without changing the geometry of the area that triggers the account settings dropdown
I was planning to reduce the value to the minimum required and could not find other way to make it work without cascade changes to other styles related to the sidebar header